### PR TITLE
[fix](default value): enable HLL_EMPTY_DEFAULT_VALUE

### PR DESCRIPTION
**Pull request by my-vegetable-has-exploded**
## Proposed changes

Issue Number: refers #34228

The goal of this pr is to make it possible to use `HLL_EMPTY()` to fill missing hll column.  As shown in `test_default_hll.groovy`,  if we don't specify hll column, it will use HLL_EMPTY to fill it.

https://github.com/apache/doris/blob/f180d90bb8db1af8bee43e095de55cfe72e95ecc/fe/fe-core/src/main/java/org/apache/doris/analysis/ColumnDef.java#L362-L367

Existing analysis will add HLL_EMPTY_DEFAULT_VALUE as default value for hll column, but some relative problem would occurs if we didn't pass `hll_hash(x)` ... for this column. 

This pr add  `defaultValueExprDef` for HLL_EMPTY_DEFAULT_VALUE (whick is used in streamload mode) and make default hll column pass the analysis for `NativeInsertStmt` and `Load`.

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by my-vegetable-has-exploded** [comment]
run buildall

**Comment by my-vegetable-has-exploded** [comment]
ptal @Yukang-Lian @BiteTheDDDDt

**Review by Yukang-Lian — APPROVED** [review]
LGTM

File: fe/fe-core/src/main/java/org/apache/doris/analysis/NativeInsertStmt.java [hunk 1]
```diff
@@ -579,7 +579,8 @@ private void analyzeSubquery(Analyzer analyzer, boolean skipCheck) throws UserEx
             }
             // hll column must in mentionedColumns
             for (Column col : targetTable.getBaseSchema()) {
-                if (col.getType().isObjectStored() && !mentionedColumns.contains(col.getName())) {
+                if (col.getType().isObjectStored() && !col.hasDefaultValue()
```
Comment (my-vegetable-has-exploded) [inline fe/fe-core/src/main/java/org/apache/doris/analysis/NativeInsertStmt.java:582]: If column has defaultValue, skip this check.

File: regression-test/suites/correctness_p0/test_default_hll.groovy [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+suite("test_default_hll") {
+    sql "SET enable_nereids_planner=true;"
+    sql "SET enable_fallback_to_original_planner=false;"
```
Comment (morrySnow) [inline regression-test/suites/correctness_p0/test_default_hll.groovy:20]: remove these two line, then
1. the default conf is enable_nereids_planner=true and enable_fallback_to_original_planner=false now
2. we could test both legacy planner and nereids planner with one test case according to cluster conf.

Comment (my-vegetable-has-exploded) [reply regression-test/suites/correctness_p0/test_default_hll.groovy]: Get！ Thanks.

**Comment by my-vegetable-has-exploded** [comment]
run buildall

**Comment by my-vegetable-has-exploded** [comment]
@morrySnow thanks for review. Any more comments?

**Review by ByteYue — APPROVED** [review]

**Comment by Yukang-Lian** [comment]
run performance

**Review by morrySnow — APPROVED** [review]

**Comment by my-vegetable-has-exploded** [comment]
thanks all！

**Pull request merged**